pkg/virt-handler/device-manager/pci_device_test.go

```diff
@@ -156,7 +156,7 @@ pciHostDevices:
 			deviceController.updatePermittedHostDevicePlugins(),
 		)
 		Expect(len(enabledDevicePlugins)).To(Equal(1), "a device plugin wasn't created for the fake device")
-		Expect(len(disabledDevicePlugins)).To(Equal(0))
+		Expect(disabledDevicePlugins).To(HaveLen(0))
```

Comment:
Nice! One thing to think about is if possible, it's also nice to explain the intent behind the comparison/check being done--like the line above.

This is a vast improvement over the error message "expected 1 to equal 0", but could in theory be even better with a human readable message explaining the there shouldn't be any disabled device plugins.

Reply:
Yup, totally agree!

Much better to add these as tests are being written, since sometimes it's hard to know the original intent behind these checks :)

I've added some explanations now :+1: